Integrations/integration-Zendesk.yml

```diff
@@ -0,0 +1,462 @@
+        }
+
+        return JSON.stringify(incidents);
+    }
+
+
+    switch (command) {
+        // This is the call made when pressing the integration test button.
+        case 'test-module':
+            var res = sendRequest('GET', 'users/me.json');
+
+            if (res.user.name !== 'Anonymous user' &&
+                res.user.id) {
+                return 'ok';
+            }
+
+            return JSON.stringify(res);
+
+        case 'zendesk-create-ticket':
+            return createTicket();
+
+        case 'zendesk-list-tickets':
+            return listTickets();
+
+        case 'zendesk-update-ticket':
+            return updateTicket();
+
+        case 'zendesk-ticket-details':
+            return getTicketDetails();
+
+        case 'fetch-incidents':
+            return fetchIncidents();
+        default:
+    }
+  type: javascript
+  commands:
+  - name: zendesk-create-ticket
+    arguments:
+    - name: subject
+      description: The subject of the ticket
+    - name: requester_id
+      description: The numeric ID of the user asking for support through the ticket
+    - name: submitter_id
+      description: The numeric ID of the user submitting the ticket
+    - name: assignee_id
+      description: The numeric ID of the agent to assign the ticket to
+    - name: group_id
+      description: The numeric ID of the group to assign the ticket to
+    - name: body
+      required: true
+      description: Comment body
+    - name: public
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Does the body public comment
+      defaultValue: "true"
+    - name: type
+      auto: PREDEFINED
+      predefined:
+      - problem
+      - incident
+      - question
+      - task
+      description: Allowed values are problem, incident, question, or task
+    - name: priority
+      auto: PREDEFINED
+      predefined:
+      - urgent
+      - high
+      - normal
+      - low
+      description: Allowed values are urgent, high, normal, or low
+    - name: tags
+      description: An array of tags to add to the ticket
+      isArray: true
+    - name: external_id
+      description: An ID to link Zendesk Support tickets to local records
+    - name: forum_topic_id
+      description: The numeric ID of the topic the ticket originated from, if any
+    - name: problem_id
+      description: For tickets of type "incident", the numeric ID of the problem the
+        incident is linked to, if any
+    - name: due_at
+      description: For tickets of type "task", the due date of the task. Accepts the
+        ISO 8601 date format (yyyy-mm-dd)
+    - name: via_followup_source_id
+      description: The id of a closed ticket for a follow-up ticket.
+    - name: collaborator_ids
+      description: An array of the numeric IDs of agents or end-users to CC on the
+        ticket. An email notification is sent to them when the ticket is created
+      isArray: true
+    - name: collaborators
```

Comment:
It is from their documentation. For my understanding, you can write here directly the emails. I checked with emails and it worked.